### DISCUSSION TITLE
Automatic file printing

### DISCUSSION THREAD

**Discussion by Whikron** [sumatrapdfreader/sumatrapdf#3503] [Q&A]
Hi,
I've got a problem. 

When printing quickly (printing from the context menu), the default settings of Sumatra are not taken into account. Is there any advice for this?  The file prints with the default printer settings

**Comment by GitHubRulesOK** [comment]
Context Menu ? internally does not offer print that's an icon or hot key CTRL +P etc

Externally SumatraPDF if default handler might be invoked with OS Print verb (mine is not even though SumatraPDF is installed, and unsure what removed that ability) but that's a system controlled registry invocation that is outside SumatraPDF's control so as you suggest will depend more heavily than normal on system default settings. You would probably need to customise the registry for your own preferences.

An alternative I might on occasions use, would be to prepare desired print settings in a batch file and right click send files or folders to the batch file for calling SumatraPDF or more likely another application (perhaps GhostScript printing).
